std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -8,7 +8,7 @@ version(unittest) import std.conv : text;
 // KRRegion
 /**
 $(D KRRegion) draws inspiration from the $(LINK2
-std_experimental_allocator_region.html, region allocation strategy) and also the
+std_experimental_allocator_building_blocks_region.html, region allocation strategy) and also the
```

Comment:
Oh, didn't realize that it has custom text. Use `MREF_ALTTEXT` then; text goes first:

    $(MREF_ALTTEXT region allocation strategy, std,experimental,allocator,building_blocks,region)

(I had also made a mistake with `building_blocks`. The underscore is part of the name there.)